src/watertap_contrib/reflo/property_models/fo_draw_solution_properties.py

```diff
@@ -0,0 +1,815 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2023, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+
+"""
+This module contains a basic property package for simple water treatment models.
+Volumetric flow and component concentration are used to determine mass flow. 
+"""
```

Comment:
Update this comment (I don't think it is accurate).

Reply:
Thanks, updated!